docs/source/conf.py

```diff
@@ -277,7 +278,8 @@
 html_static_path = ['_static']
 
 # Custom fixes to the RTD theme
-html_css_files = ['theme_overrides.css']
+html_css_files = ['theme_overrides.css',
+                  'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css']
```

Comment:
Is this needed if you included the svg files itself?

Reply:
It's needed for the dropdown in _contributing.rst_ - but I can just as well omit the icons (and this fix in the theme) in both dropdowns, will be fine either way.

Reply:
Ah, I see. I think it is fine to keep them. 
In this dropdown, I suppose you cannot easily use a downloaded image as you did for the others?

Now, the theme itself should normally also already include fontawesome. So I would still think that it would even work as is without the above line. For example, if you look at [view-source:https://alenkaf.github.io/arrow-dev-docs-preview/html/developers/contributing.html](view-source:https://alenkaf.github.io/arrow-dev-docs-preview/html/developers/contributing.html), there is already some other `/vendor/fontawesome/..` css included as well.

Reply:
Ok, I understand. Will try without and see what happens ...

Reply:
It finds `hand-point-up` but not the `pencil` icon 🤦‍♀️ 
Looking at it I see is actually nicer without the icons in the dropdown so I will remove them.